layouts/shortcodes/docs/languages/exporters/prometheus-setup.md

```diff
@@ -0,0 +1,48 @@
+## Prometheus
+
+To send your metric data to [Prometheus](https://prometheus.io/), you can either
+[enable Prometheus' OTLP Receiver](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver)
+and use the [OTLP exporter](#otlp) or you can use the
+[`PrometheusHttpServer`](https://javadoc.io/doc/io.opentelemetry/opentelemetry-exporter-prometheus/latest/io/opentelemetry/exporter/prometheus/PrometheusHttpServer.html),
+a `MetricReader`, that starts an HTTP server that will collect metrics and
+serialize to Prometheus text format on request.
+
+### Backend Setup {#prometheus-setup}
+
+<div class="alert alert-info" role="alert"><h4 class="alert-heading">Note</h4>
+
+If you have Prometheus or a Prometheus-compatible backend already set up, you
+can skip this section and setup the [Prometheus](#prometheus-dependencies) or
+[OTLP](#otlp-dependencies) exporter dependencies for your application.
+
+</div>
```

Comment:
We should explore ways to be able to use shortcodes here (most likely in a template).

Reply:
We'll track this via:

- #5726